src/DataExplorer/actionHelper.js

```diff
@@ -80,30 +120,11 @@ export const downloadManifest = async (
     || !hasKeyChain(arrangerConfig, 'manifestMapping.referenceIdFieldInResourceIndex')) {
     throw MSG_DOWNLOAD_MANIFEST_FAIL;
   }
-  const resourceIDList = (await queryDataByIds(
+  const manifestJSON = await createManifestByFilter(
```

Comment:
Could move `checkArrangerGraphqlField` into your new `createManifestByFilter` function since this same check logic is also used in `exportToWorkspace` function.

Reply:
yep! fixed in https://github.com/uc-cdis/data-portal/pull/538/commits/9ce4f53422a1f0611b6fa5afd0b11a1dc76c8a2f